airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1043,6 +1043,114 @@ definitions:
         additionalProperties: true
       documentation_url:
         type: string
+      auth_flow:
```

Comment:
is there anywhere else where this object differs from the protocol spec field already? if not, could we just keep the name the same and not have to maintain a translation layer?

Reply:
we can keep the same name, but we'll still need the translation layer in `Spec` for `url_documentation` -> `urlDocumentation` and to to go from one enum type to another for the auth type.